File: Java/program124.java
import java.util.*;


class program124
{
    public static void main(String A[])
    {
        int iCnt = 0, iLength = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number of elements ");
        iLength = sobj.nextInt();

        int [] Arr = new int [iLength];
        

        

        System.out.println("Enter elements :");

        for(iCnt = 0; iCnt < iLength ; iCnt++)
        {
            Arr[iCnt] = sobj.nextInt() ;
        }


    }
}

File: C++/program124.cpp
#include<iostream>
using namespace std;

int main()
{
    int iLength = 0;
    int *ptr = NULL;

    cout<<"Enter number of elements "<<'\n';
    cin>>iLength;

    ptr = new int[iLength];

    if(NULL == ptr)
    {
        cout<<"Unable to allocate memory"<<'\n';
        return -1;
    }


    delete [] ptr;
    return 0;
}